README.MARCJOSEPH.md

```diff
@@ -0,0 +1,31 @@
+# Car Rules 
+general Usage Rules
+
+1.	The vehicle can only go to church, school, and restaurants. All other destinations require permission.
+2.	Passengers are limited to Katrina Wills and family. Any other passengers require permission.
```

Comment:
remove Katrina -- and make copy "Passenger are not currently allowed. Any passenger requires a pull request for each day/time"